src/compiler/factory.ts

```diff
@@ -2214,21 +2210,113 @@ namespace ts {
         return tag;
     }
 
-    /* @internal */
     export function createJSDocComment(comment?: string | undefined, tags?: NodeArray<JSDocTag> | undefined) {
         const node = createSynthesizedNode(SyntaxKind.JSDocComment) as JSDoc;
         node.comment = comment;
         node.tags = tags;
         return node;
     }
 
-    /* @internal */
     function createJSDocTag<T extends JSDocTag>(kind: T["kind"], tagName: string): T {
         const node = createSynthesizedNode(kind) as T;
         node.tagName = createIdentifier(tagName);

```

Comment:
this is missing an assignment to `node.comment`

Reply:
That not my fault, 
/cc: @DanielRosenwasser